traffic_ops/traffic_ops_golang/cdni/shared.go

```diff
@@ -200,6 +201,44 @@ func PutHostConfiguration(w http.ResponseWriter, r *http.Request) {
 	api.WriteAlerts(w, r, http.StatusAccepted, alerts)
 }
 
+// DeleteConfiguration deletes CDNi configuration for ucdn specified in JWT
+func DeleteConfiguration(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	if inf.Config.Cdni == nil || inf.Config.Secrets[0] == "" || inf.Config.Cdni.DCdnId == "" {
```

Comment:
If `inf.Config.Secrets` has a length of 0, the second condition check will cause a panic. Could you pls add a check for that as well?